### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by studeba** [jotaen/xit#50] [Other]
It would be great to be able to add/check-off the checkboxes with keyboard shortcuts

**Comment by jotaen** [comment]
You mean when looking at a `.xit` file in an editor? That would have to be implemented in the editor plugin you are using. For example, in the [one for Sublime Text](https://packagecontrol.io/packages/xit!) this is supported via the shortcut `ctrl+shift+x`.
